### DISCUSSION TITLE
HOW TO GET THE WINDOW MAXIMIZED/ZOOMED (CustomTkinter, CTk):

### DISCUSSION THREAD

**Discussion by Wasi-Genius** [TomSchimansky/CustomTkinter#1876] [General]
So I was trying to find a solution on how to get a Custom Tkinter window maximized and I finally found a solution on overflow:  

<img width="491" alt="image" src="https://github.com/TomSchimansky/CustomTkinter/assets/103324585/aef6512a-9938-4630-8058-eda52d5ea426">

Also, here is my working code for reference: 

```
from customtkinter import *
from tkinter import filedialog as fd
from PIL import Image

set_default_color_theme('dark-blue')

root = CTk()
root.title('Image Viewer')
root.iconbitmap(
    'C:\PythonProjectsWasiG\GUI Python\ImageViewerEditor\ImageViewerIcon.ico')
root.after(1, root.wm_state, 'zoomed')
```

**Comment by Maimx** [comment]
Or you can also use an additional library.
(It took me a while to find this library)

Install the library:
`pip install pygetwindow`

Code:
```
import pygetwindow as gw

root = tk.Tk()

def maximize_window():
       window = gw.getActiveWindow()
       window.maximize()

root.after(1000, maximize_window)
root.mainloop()
```

**Reply by therealzakie** [reply]
This isn't very multi-platform friendly though :(
